Update to Python 3

Nionor opened the pull request:
It's kinda sloppy since I just wanted something I could use, but maybe it helps on the way.

SchapplM commented:
looks good to me. The alternative would be not having the addon any more.
I also tested with Dropbox synchronization, which also worked.
I will merge this as soon as the buggalo addon works.

Nionor commented:
When I changed the code for the buggalo email I also changed it for dropbox, but since that worked before maybe it broke now? I can't test it right now, but maybe just change it back if that's the case.
